### ISSUE TITLE
Minor rendering bug for clickable `docs serve` output

### ISSUE THREAD

**Issue by aaronsteers** [fishtown-analytics/dbt#2027]
### Describe the bug

When running `dbt serve` the printout of the url where the docs are served is rendering with a period ('.') immediately adjacent to the link. This is a minor issue but it prevents link detection from working correctly by terminals (VS Code in my case) which automatically detect URLs and render them as hyperlinks. In this case, the output is correctly parsed as a URL but because the URL includes a '.' at the end, it reaches a dead end (equivalent to pasting `http://localhost:8081.` in chrome address bar).

```
To access from your browser, navigate to http://localhost:8080.
```

### Steps To Reproduce

1. Run `dbt docs serve`

### Expected behavior

Ideally, the printout would be rendered as:

```
To access from your browser, navigate to: http://localhost:8080
```

### Screenshots and log output

```
Running with dbt=0.13.0
Serving docs at 0.0.0.0:8080
To access from your browser, navigate to http://localhost:8080.
Press Ctrl+C to exit.
```

### System information
**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [ ] bigquery
- [ ] snowflake
- [ ] other (specify: ____________)

**The output of `dbt --version`:**
```
Running with dbt=0.13.0
```

**The operating system you're using:**

**The output of `python --version`:**

**Comment by aaronsteers** [comment]
PR created here: https://github.com/aaronsteers/dbt/pull/1

**Comment by drewbanin** [comment]
Thanks @aaronsteers - I buy this :)

Can you open the PR you made against this repo? If you set the base branch to `dev/barbara-gittings`, then I'd be happy to merge it post-haste :)

**Comment by aaronsteers** [comment]
Happy to! I'll rebase against that branch (`dev/barbara-gittings`), reopen the PR in this repo, and send this over.

**Comment by drewbanin** [comment]
hey @aaronsteers - are you still able to open up that PR against this repo? Would be good to get this fix merged for v0.16.0!

**Comment by aaronsteers** [comment]
@drewbanin - Yes - sorry to delay this. I'll try to get this done today against `dev/barbara-gittings` - unless I should use a different target (just lmk).

**Comment by aaronsteers** [comment]
Closed and replaced with #2131
